style.css

```diff
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lexend+Deca&display=swap');
+
+:root {
+  --grey: #979797;
+  --dark: #1B1937;
+  --white: #FFFFFF;
+  --purple: #AB5CDB;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+body {
+  background-color: black;
+}
+
+main, footer {
+  width: 327px;
+  display: flex;
+  margin: 0 auto;
+  margin-top: 30px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h1, dd {
+  line-height: auto;
+  text-align: center;
+  color: var(--white);
+  letter-spacing: 0.5px;
+}
+
+p, dt {
+  color: var(--grey);
+  text-align: center;
+  letter-spacing: 0.5px;
+}
+
+.img {
+  overflow: hidden;
+  margin-bottom: -4px;
+  border-radius: 8px 8px 0 0;
+  background-color: var(--purple);
+}
+
+img {
+  opacity: 0.4;
+  height: 240px;
+  object-fit: cover;
+}
+
+.info-wrapper {
+  padding: 20px;
+  border-radius: 0 0 8px 8px;
+  background-color: var(--dark);
+}
+
+h1 {
+  font-size: 28px;
+  font-weight: bold;
+  line-height: 32px;
+}
+
+.info-wrapper  {
+  padding-top: 20px;
+}
+
+.info > p {
+  font-size: 15px;
+  line-height: 1.5;
+  margin-top: 20px;
+}
+
+span {
+  color: var(--purple);
+}
+
+.stats-item {
+  display: flex;
+  flex-direction: column-reverse;
+  margin-top: 30px;
+}
+
+.stats-item .number {
+  font-size: 24px;
+}
+
+dt {
+  font-size: 12px;
+  line-height: 25px;
+  text-transform: uppercase;
+  font-family: 'Lexend Deca', sans-serif;
+}
+
+footer div {
+   margin-bottom: 15px;
+}
+
+a {
+  color: var(--purple);
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--grey) ;
+}
+
+@media (min-width: 400px) {
+  body {
+    top: 20%;
+    left: 20%;
+    position: fixed;
+  }
+  main {
+    width: 1110px;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row-reverse;
+  }
+
+  h1, p {
+    text-align: start;
+  }
+
+  .img {
+    width: 100%;
+    height: 446px;
+    overflow: hidden;
+    margin-bottom: 0px;
+    border-radius: 0 8px 8px 0;
+  }
+
+  img {
+    height: 446px;
+  }
+
+  .info-wrapper {
+    display: flex;
+    height: 446px;
```

Comment:
When you use static width or height it is an indication that you might have issues with responsiveness. Try min-height instead or alternative styling using flexbox